src/pages/q/[[...all]].tsx

```diff
@@ -100,10 +100,17 @@ const SearchIndex: any = ({
       !selectedTopics.includes('undefined')
     ) {
       const newTopic = first<string>(selectedTopics)
-
       try {
         if (newTopic) {
-          const cachedTag = tagCacheLoader(newTopic)
+          let cachedTag
+          if (canLoadSanityTag(newTopic)) {
+            cachedTag = await loadTag(newTopic)
+          } else {
+            cachedTag = tagCacheLoader(newTopic)
+          }
+
+          setTopic(cachedTag)
+
```

Comment:
Turns out we are still having issues here. Going to reconvene with Zac at a later date and get this figured out.